src/jsonpointer.js

```diff
@@ -155,6 +190,55 @@
     };
   }
 
+  function createPointerSetter(target) {
+    return function(pointer, opt_targetvalue) {
+
+      if( pointer === '' ) {
```

Comment:
Please keep the code style consistent.
There is always a whitespace between `if` and opening brace,
and there are no spaces before and after expression between braces.

So this line should look like this: `if (pointer === '') {`.